src/table/body-cell/td-selection-element.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import clsx from 'clsx';
+import React, { KeyboardEventHandler } from 'react';
+import SelectionControl from '../selection-control/';
+import { TableProps } from '../interfaces.js';
+import { TableTdElementProps } from './td-element.js';
+import { useVisualRefresh } from '../../internal/hooks/use-visual-mode';
+import styles from './styles.css.js';
+
+type TableBodySelectionCellProps = Omit<
```

Comment:
The Omit declaration is difficult to read. Would it be simpler to declare the props explicitly? Also, the 'tdRef' does not exist.